receiver/kubeletstatsreceiver/config.go

```diff
@@ -105,3 +118,10 @@ func (cfg *Config) Unmarshal(componentParser *confmap.Conf) error {
 
 	return nil
 }
+
+func (cfg *Config) Validate() error {
```

Comment:
We need to add a `config_test` scenario for this validation

Reply:
Sure, I have added one validation test.